util.go

```diff
@@ -82,11 +82,13 @@ func DefaultTableSummary() map[int]func(io.Writer, int) (int, error) {
 //               |                           |
 //
 type LineStyle struct {
-	Top  [4]rune
-	Mid  [4]rune
-	Row  [4]rune
-	Wrap [4]rune
-	End  [4]rune
+	Top          [4]rune
+	Mid          [4]rune
+	Row          [4]rune
+	Wrap         [4]rune
+	End          [4]rune
+	EmbedColumns bool
```

Comment:
Please rename `EmbedColumns` to `Inline` and add a comment indicating what it is for. Specifically:

```go
~     // Inline causes the column header names to be rendered "inline" with the
+     // top table line.
+     Inline bool
```